Assets/HoloToolkit/Utilities/Scripts/Inspectors/MRTKEditor.cs

```diff
@@ -28,7 +28,17 @@ public class MRTKEditor : Editor
     {
         #region static vars
         // Toggles custom editors on / off
-        public static bool ShowCustomEditors { get; private set; }
+        public static bool ShowCustomEditors
+        {
+            get
+            {
+                return EditorPrefs.GetBool("MRTK_Show_MRTK_Editors");
```

Comment:
Please use the `EditorPrefsUtility` class

Reply:
I can't find any reference EditorPrefsUtility in the repository.

Reply:
https://github.com/Microsoft/MixedRealityToolkit-Unity/blob/master/Assets/HoloToolkit/Utilities/Scripts/Editor/EditorPrefs.cs

The name of the class is incorrect

Reply:
Got it. Since that's an Editor class I can't use it directly but I've mimicked the technique inside of MRTKEditor.